### PR TITLE
Create QingGO.yaml

### PR DESCRIPTION
**Pull request by QingGo**

### CODE REVIEW

File: _data/signed/QingGO.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name: 曾颖清 (QingGO)
```
Comment (tmwstw7) [inline _data/signed/QingGO.yaml]: Hi there, I suppose your name in Chines fails CI\CD. I'm not trying to harass or smth. but currently everybody get this error
image

Comment (Lesik) [reply _data/signed/QingGO.yaml]: In that case in my opinion the appropriate solution is to fix the CI script, not change the name.

Comment (Lesik) [reply _data/signed/QingGO.yaml]: (Both yaml and Jekyll fully support Unicode.)

Comment (QingGo) [reply _data/signed/QingGO.yaml]: I find that my PR did not pass the check. so i try to use PinYin to write my name in this PR, or maybe the check script should by modify.

Comment (tmwstw7) [reply _data/signed/QingGO.yaml]: I think @Lesik is wright, there might be some problems with CI. Also I double-checked the verification script, it seems to work fine.

Comment (QingGo) [reply _data/signed/QingGO.yaml]: glad to hear that

**Pull request merged**